scripts/performance-report.ts

```diff
@@ -102,6 +102,14 @@ export async function generatePerformanceReport(path: string) {
         'process_cpu_seconds_total',
         'playwright_total_time'
     ]);
+
+    // TODO Post process values:
+    // if label ends with _X
+    //  take current as label
```

Comment:
Sort and remove last element?

Reply:
Hi Matthew, thanks for your comment! However, I'm not entirely sure what you mean?
I've pushed a first feature-complete version of the workflow that performs ten runs of the measurement and then prints average and best of 10 per metric.
Please feel free to comment! Thank you!

Reply:
I was suggesting a way to lazily handle rejecting the worst case outlier. I looked at your new code, It's pretty much perfect for that.

Considerations by the way:
* Worst case execution time can be due to being pre-empted.
* Best case can be due to cache. Since you rebuild a docker image each time, I think cache would be negligeable. So no need to remove the best case.

Reply:
Thanks for the clarification! Good idea!
I was thinking that maybe the median may even be a better choice as it would remove the effect of outliers altogether. I've changed the implementation accordingly in b26a5e7. Please let me know if you disagree. Thanks!

Reply:
Low outliers can be an issue. I would worry much less about them than the high outliers.
Basically functions duration tend to look like a chi2 function.
![image](https://github.com/eclipse-theia/theia-e2e-test-suite/assets/419535/487b1beb-3220-4bb1-8f55-2caa3e385330)
Here is an example of typical IRQ latency.

![irq-example](https://github.com/eclipse-theia/theia-e2e-test-suite/assets/419535/204ef9f2-ea59-4bb4-a685-5a9a163329f7)

Rejecting the low end could be interesting as it would give more "consistent" data, but it reflects the behaviour of the software much more than the slow data that often is slow due to external interference.

TL;DR: Outside programs cannot interfere with your code and make it faster. If we want to reject interference, we reject the slow outliers. Fast outliers are often due to cache or failures. That is a separate but valid concern.

Reply:
Thank you for the input! Interesting!
Does that mean you would prefer an average that excludes best and worst case? Or should we keep the median, which should be rather similar than average rejecting best and worst?

Reply:
I would prefer the average excluding worst offenders at first. If we need to update the post-processed algorithm later, it can be done though as you retain the data.

I have a very LIGHT preference though and as long as the worst offenders are rejected, I'm fine.